cmd/main.go

```diff
@@ -152,6 +167,7 @@ func setupSignalHandler() (stopCh <-chan struct{}) {
 		close(stop)
 		<-c
 		os.Exit(1) // second signal. Exit directly.
+		klog.Info("exiting")
```

Comment:
`exiting` may not make much sense when viewed in a log file without any context. Maybe a bit more information here would be useful.

Reply:
this ended up not proving useful ... I'm just going to remove, thanks.